app/stylesheets/config/type.scss

```diff
@@ -1,3 +1,27 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700,800,300,600);
 @mixin no-margin() {
   margin: 0;
 }
+
+// basic text style in top and bottom tabs
+@mixin tab-text() {
+  font-size: 12px;
```

Comment:
Lets put `font-size` as a variable. By "top and bottom tabs" do you mean the filters? In the screenshot they look like `font-weight: normal` I think, so maybe it's better to just use variables instead of a mixin